roles/valheim/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+#########################################################################
+# Title:            Sandbox: valheim                                    #
+# Author(s):        scarabyte                                           #
+# URL:              https://github.com/saltyorg/Sandbox                 #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+
+- name: Remove existing Docker container
+  ansible.builtin.include_tasks: "{{ resources_tasks_path }}/docker/remove_docker_container.yml"
+
+- name: Add DNS record
+  ansible.builtin.include_tasks: "{{ resources_tasks_path }}/dns/tasker.yml"
+  vars:
+    dns_record: "{{ lookup('vars', role_name + '_dns_record') }}"
+    dns_zone: "{{ lookup('vars', role_name + '_dns_zone') }}"
+    dns_proxy: "{{ lookup('vars', role_name + '_dns_proxy') }}"
+
+- name: Add SRV record
```

Comment:
This whole block seems to be copied from minecraft without ever editing it making this kind of useless. Does this Valheim need the SRV record? If it does then adapt it as needed.